libvirt.go

```diff
@@ -918,6 +933,39 @@ func (l *Libvirt) Version() (string, error) {
 	return versionString, nil
 }
 
+// Shutdown shuts down a domain
```

Comment:
This comment can be improved and should be a full sentence. Add a note on the fact that if no flags are provided (`flags == 0`), libvirt will chose the best available method.